contents/product/collaboration.mdx

```diff
@@ -23,6 +23,171 @@ featuredImage: ../images/apps/collaboration.png
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="bg-gray-accent bg-opacity-50 text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td>Self-hostable</td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="bg-gray-accent bg-opacity-50 text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td>Cloud hosting</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="bg-gray-accent bg-opacity-50 text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td>Product analytics</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="bg-gray-accent bg-opacity-50 text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td><strong>Collaboration</strong></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td className="bg-gray-accent bg-opacity-50">&nbsp;</td>
+        </tr>
+        <tr>
+            <td>User-level permissions</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="bg-gray-accent bg-opacity-50 text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+        <tr>
+            <td>Project-level permissions</td>
```

Comment:
What does project-level mean vs. user-level? What do those pertain to?

Reply:
With user-level permissions, you can set a user to be either an owner, editor or viewer (for example). This role applies across all projects. 

With project-lever permissions, you can set a user to be an owner of one project, an editor of another and a viewer of a third.

Amplitude, for example, [has the concept of both](https://help.amplitude.com/hc/en-us/articles/229672228-Set-and-edit-user-permissions) and the use-case is typically for creating teams within an org. On PostHog, we seem to have user permissions and private projects, but not project-level permissions.

Reply:
I think it's fair to say private projects are a basic form of project-level permissions, so maybe this could be like an orange "partial" marker?
On a separate note, we also have dashboard-level permissions, which I don't think are called out here (TBH not sure what the competition has in that regard)?

Reply:
Eh, you've talked me round to just turning this into a tick tbh. Users can have private projects, or collaborative projects and if they want collaborative projects then they can choose user permissions. There's no mismatch of function, only terminology. 

There is some difference in how dashboard-level permissions are handled, but all of the competitors have this and I've added it in too.